go-controller/pkg/util/ovs.go

```diff
@@ -941,3 +941,27 @@ func DetectCheckPktLengthSupport(bridge string) (bool, error) {
 
 	return false, nil
 }
+
+type DbProperties struct {
+	AppCtl        func(timeout int, args ...string) (string, string, error)
+	DbAlias       string
+	DbName        string
+	ElectionTimer int
+}
+
+func PropertiesForDB(db string) *DbProperties {
+	if strings.Contains(db, "ovnnb") {
+		return &DbProperties{
+			ElectionTimer: int(config.OvnNorth.ElectionTimer) * 1000,
+			AppCtl:        RunOVNNBAppCtlWithTimeout,
+			DbName:        "OVN_Northbound",
+			DbAlias:       db,
+		}
+	}
+	return &DbProperties{
+		ElectionTimer: int(config.OvnSouth.ElectionTimer) * 1000,
+		AppCtl:        RunOVNSBAppCtlWithTimeout,
+		DbName:        "OVN_Southbound",
+		DbAlias:       db,
+	}
+}
```

Comment:
Given the broader scope placement of this utility, should we change the name `DbProperties` to `OvsDbProperties` and implement `PropertiesForDB` checking explicitly for `ovnsb` and handling with an error unexpected dbs? WDYT?

Reply:
Actually maybe narrowing down the names a bit more might be better. Would `OvsDbAppCtlSetupProperties`/`GetOvsDbAppCtlSetupPropertiesForDb` be reasonable?

Reply:
I am not sure about `OvsDbAppCtlSetupProperties` since this struct is not only about appCtl, and can be extended at some point (and it has `AppCtl` field, so including that to the name may be redundant), but I like the idea of renaming (didn't come to my mind) - what about `OvsDbProperties`+ `GetOvnDbProperties`?

Reply:
I like your suggestions - I think renaming to `GetOvsDbProperties` and adding error handling with some comments is the best option - so please check the latest changes (still open to further comments - my wording for error messages/comments may not be the best :D)